### DISCUSSION TITLE
When installing new apps, they do not show up until  you reboot

### DISCUSSION THREAD

**Discussion by binarykitchen** [end-4/dots-hyprland#645] [General]
When I, for example, run `pacman -S google-chrome` and then hit the super button, type it, it doesn't appear.

Is there no refresh mechanism? Always have to reboot hyprland to make them show up. Isn't there some automatic solution?

**Comment by end-4** [comment]
it's a known AGS issue
you can restart AGS by pressing Ctrl+Super+R and it should show up

**Comment by binarykitchen** [comment]
Alright - it's just a pain but understand. Peace.
